azkaban-exec-server/src/test/java/azkaban/execapp/StatusTestUtils.java

```diff
@@ -0,0 +1,23 @@
+package azkaban.execapp;
+
+import azkaban.executor.ExecutableNode;
+import azkaban.executor.Status;
+
+public class StatusTestUtils {
+
+  public static void waitForStatus(final ExecutableNode node, final Status status) {
```

Comment:
Could you use this method to replace the same logic in 
https://github.com/HappyRay/azkaban/blob/96ad15a98af88adb1c17b7486802ed016bd109f1/azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTestBase.java#L120
It can be done in a separate PR.